pstore/kafka/kafka.go

```diff
@@ -5,9 +5,9 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/Symantec/scotty/pstore"
+	"github.com/Symantec/scotty/pstore/lmm"
```

Comment:
Why is this called lmm rather than kafka? The former is an internal, non-generic name.